coalib/bears/Bear.py

```diff
@@ -47,6 +49,21 @@ class Bear(Printer, LogPrinter):
     >>> class SomeBear(Bear):
     ...     LANGUAGES = "Java"
 
+
+    To indicate the requirements of the bear, give to the ``REQUIREMENTS``
+    tuple instances of the ``PackageRequirement`` class:
```

Comment:
hmm... give to a tuple ... rather saying "Assign ``REQUIREMENTS`` a tuple with instances of ..."?